site.properties

```diff
@@ -242,7 +249,7 @@ pql.lola.dir = /wherever/you/keep/lola
 
 # The PQL databases (Postgres only required for VSM similarity search)
 
-pql.mysql.url      = jdbc:mysql://localhost/apromore?autoReconnect=true
+pql.mysql.url      = jdbc:mysql://apromore-mysql.apromoresso.net/apromore?autoReconnect=true
```

Comment:
This shouldn't be the default.

Reply:
@raboczi I will reverse the comments, and leave localhost:3306 as the default and apromore-mysql.apromoresso.net with an annotated comment